spec/doc_builder_static_data.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 # Test data for autotest
-module TestData
```

Comment:
At different times added two files on for static data 

I'm fixing this problem in this commit https://github.com/ONLYOFFICE/doc-builder-testing/pull/841/commits/66d776edc8b87f9ab6a4788c16bebfc4cb4aab5b, but it's hard to isolate outside of this commit because there are a lot of dependencies

I think in the future I'll split the changes into different commits for easier checking and working

Reply:
Changes have been rolled back 

https://github.com/ONLYOFFICE/doc-builder-testing/pull/841/commits/98ed4b7206b2b9e3eaf35dfe8b1d6823d4cdf5f8